src/sgame/sg_buildable.cpp

```diff
@@ -775,10 +775,13 @@ static bool CompareBuildablesForPowerSaving(Entity* a, Entity* b)
 void G_UpdateBuildablePowerStates()
 {
 	std::list<Entity*> buildables;
+	std::list<Entity*> unpoweredBuildables;
```

Comment:
Why are we using doubly linked lists? I don't see anything where a vector wouldn't suffice.

Also these declarations could be moved inside the loop.

Reply:
idk, maybe there is no random access and everything is accessed linearly so you never take a resize hit? Moved the declarations inside the loop.

Reply:
When everything is accessed linearly that just makes the vector more preferable. Vectors are the default choice (for a small, trivially copyable object at least) since the higher amount of allocations for a linked list is more costly than copying a contiguous block of memory a few times.

Reply:
Done.